<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-VJJTB407" ignored="1" name="Submit Order Test Data" tests="2" failures="1" timestamp="2022-11-07T13:43:46 EST" time="50.057" errors="0">
  <testcase name="submitOrder" time="16.33" classname="company.tests.SubmitOrderDataTest"/>
  <testcase name="submitOrder" time="14.788" classname="company.tests.SubmitOrderDataTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at company.tests.SubmitOrderDataTest.submitOrder(SubmitOrderDataTest.java:36)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- submitOrder -->
  <testcase name="OrderHistoryTest" time="0.0" classname="company.tests.SubmitOrderDataTest">
    <ignored/>
  </testcase> <!-- OrderHistoryTest -->
</testsuite> <!-- Submit Order Test Data -->
